test/unit/controllers/CommentController.test.js

```diff
@@ -216,4 +216,42 @@ describe('CommentController', function () {
       })
     })
   })
+
+  describe.only('createBatchFromEmailForm', () => {
```

Comment:
similarly e.g. `npm run autotest -- -f test/client/components/Login.test.js` for hylo-redux